modules/iam/groups/data.tf

```diff
@@ -0,0 +1,8 @@
+data "aws_iam_policy" "admin_access" {
+  #checkov:skip=CKV_AWS_275: Admin accounts require admin access
+  arn = "arn:aws:iam::aws:policy/AdministratorAccess"
+}
+
+data "aws_iam_policy" "power_user_access" {
+  arn = "arn:aws:iam::aws:policy/PowerUserAccess"
+}
```

Comment:
These policies are not strict enough to ensure the user must either change their password or have MFA enabled.
You are possibly have a better idea, but one of the approach was implemented in this PR https://github.com/nimblehq/infrastructure-templates/pull/188
so you can have a check for inspiration.

Reply:
Thanks for sharing, I updated it in fe1406c ✅